ppr-ui/src/components/mhrRegistration/YourHome/ManufacturerMakeModel.vue

```diff
@@ -261,13 +276,15 @@ export default defineComponent({
 
     return {
       hasError,
+      nameRef,
       yearRef,
       makeRef,
       modelRef,
       makeModelComboForm,
       manufactureYearRules,
       makeRules,
       modelRules,
+      manufacturerNameRules,
```

Comment:
Just to add to the above, by including computed values in the `localState` you don't need to explicitly return them here, as there are included in the spread of the `localState` 
line 290 `...toRefs(localState)`

Just some info, no action required :)